docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -1059,6 +1059,60 @@ Default is `INFO`.
 
 Print version information and exit.
 
+## Fast Sync Options 
+
+!!! important 
+    Support for fast sync is currently experimental. Fast sync is in active development. The fast sync options are available
+    but hidden on the command line. 
+
+### sync-mode
+
+```bash tab="Syntax"
+--sync-mode=FAST
+```
+
+```bash tab="Example Command Line"
+--sync-mode=FAST
+```
+
+```bash tab="Example Configuration File"
+sync-mode="FAST"
+```
+
+Specifies the synchronization mode. Default is `FULL`.
+
+### fast-sync-max-wait-time
```

Comment:
As I understand it, this flag is for debugging - do we need to document debugging flags?